### DISCUSSION TITLE
A way to replicate Aflred Workflows/Fluent Search Tasks?

### DISCUSSION THREAD

**Discussion by WildGenius** [Flow-Launcher/Flow.Launcher#1127] [Q&A] [answered]
Essentially a flowchart GUI for creating complex macros, including launching multiple apps, running commands etc. See [Fluent Search blogpost](https://www.fluentsearch.net/posts/fluent-search-tasks-nightly-test) for an example.

**Comment by jjw24** [comment, marked as answer]
Hey how's it going?

This is definitely something on the horizon for us as well. I would see it implemented in a way that allows plugins to build this feature out rather than baking it into the main app, so any plugin could build a workflow/task runner.

**Reply by WildGenius** [reply]
Nice one. I assume since it's on the roadmap already, there's no point making a feature request for it. I was asking to check whether there is already some combination of settings/plugins that would allow this.

**Reply by jjw24** [reply]
It won't be for 1.10.0 release but likely for 1.11.0. I will convert this to feature request issue.

**Reply by jjw24** [reply]
>  some combination of settings/plugins that would allow this.

The closest would be the Runner plugin, but it requires you to write your own command to run apps etc and chain them together. https://github.com/jjw24/Wox.Plugin.Runner
